src/client/main.c

```diff
@@ -0,0 +1,60 @@
+#include <getopt.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "client.h"
+#include "libhirte/common/common.h"
+#include "libhirte/common/opt.h"
+
+const struct option options[] = { { ARG_HELP, no_argument, 0, ARG_HELP_SHORT }, { NULL, 0, 0, '\0' } };
+
+#define OPTIONS_STR ARG_HELP_SHORT_S
+
+// NOLINTBEGIN(cppcoreguidelines-avoid-non-const-global-variables)
```

Comment:
Honestly, this lint option is crap, can we just disable it?

Reply:
I agree. Can you add it to [.clang-tidy](https://github.com/containers/hirte/blob/main/.clang-tidy#L23)? @mkemel

Reply:
Done, also else-after-return rules